juriscraper/opinions/united_states/federal_appellate/cadc.py

```diff
@@ -1,41 +1,22 @@
-import time
-from datetime import date
+from juriscraper.OpinionSiteLinear import OpinionSiteLinear
 
-from lxml import html
 
-from juriscraper.OpinionSite import OpinionSite
-
-
-class Site(OpinionSite):
+class Site(OpinionSiteLinear):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
```

Comment:
Can you add the typical author/ date / etc information that we normally have.  Otherwise this looks good to me.